packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -48,8 +48,9 @@
     "clean":
       "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
     "postinstall": "node ./node_modules/vscode/bin/install",
+    "pretest": "node ../../scripts/download-vscode-for-system-tests",
     "test":
-      "cross-env CODE_TESTS_WORKSPACE='../system-tests/assets/sfdx-simple' node ./node_modules/vscode/bin/test"
+      "node ../../scripts/install-vsix-dependencies dbaeumer.vscode-eslint && cross-env CODE_TESTS_WORKSPACE='../system-tests/assets/sfdx-simple' CODE_TESTS_PATH='./' node ./node_modules/vscode/bin/test"
```

Comment:
There is a dependency from LWC to the external dbaeumer.vscode-eslint. Without having this dependency, the entire test will fail.